api/spec/serializers/spree/v2/storefront/cms_section_serializer_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe Spree::V2::Storefront::CmsSectionSerializer do
+  let(:cms_section) { create(:cms_section, name: 'Test Name') }
+
+  subject { described_class.new(cms_section) }
+
+  it { expect(subject.serializable_hash).to be_kind_of(Hash) }
+
+  it do
+    expect(subject.serializable_hash).to eq(
+      {
+        data:
+          { attributes:
+            { content: nil,
+              fit: nil,
+              img_one_lg: nil,
```

Comment:
@tomurb let's please attach some files to these records to check for proper URLs